components/BurgerMenu/CollapseMenu.tsx

```diff
@@ -0,0 +1,121 @@
+import clsx from "clsx";
+import Link from "next/link";
+import { useState } from "react";
+
+import { getLinkPath } from "@/lib/menus";
+import { translate } from "@/lib/translations";
+import { MenuItemWithChildrenFragment } from "@/saleor/api";
+
+import { useRegions } from "../RegionsProvider";
+import styles from "./BurgerMenu.module.css";
+import ChebronDown from "./chevronDownIcon.svg"; // in the final version it should be imported from ui-kit package
+
+interface CollapseMenuProps {
+  data: MenuItemWithChildrenFragment;
+}
+
+function SubCollapseMenu({ data }: CollapseMenuProps) {
```

Comment:
Separate files please, it's always difficult to navigate when in some files there are single components and in some other many